plugins/modules/zos_copy.py

```diff
@@ -1806,13 +1927,34 @@ def main():
             content=dict(type='str', no_log=True),
             backup=dict(type='bool', default=False),
             backup_name=dict(type='str'),
-            model_ds=dict(type='str', required=False),
             local_follow=dict(type='bool', default=True),
             remote_src=dict(type='bool', default=False),
             sftp_port=dict(type='int', required=False),
             ignore_sftp_stderr=dict(type='bool', default=False),
             validate=dict(type='bool'),
             volume=dict(type='str', required=False),
+            type=dict(
+                arg_type='str',
+                default='BASIC',
+                choices=['KSDS', 'ESDS', 'RRDS', 'LDS', 'SEQ', 'PDS', 'PDSE', 'LIBRARY', 'BASIC', 'LARGE', 'MEMBER', 'HFS', 'ZFS'],
```

Comment:
May need to reduce some of the options depending how the above comment/discussion ends.

Reply:
I did.  The other values were carry-overs from zos data_set, but probably don't apply here.